test/tests/operations/Image.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Image operation tests.
+ *
+ * @author tlwr [toby@toby.codes]
+ *
+ * @copyright Crown Copyright 2017
+ * @license Apache-2.0
+ */
+import TestRegister from "../../TestRegister.js";
+
+TestRegister.addTests([
+    {
+        name: "Extract EXIF: nothing",
+        input: "",
+        expectedOutput: "Found 0 tags.\n",
+        recipeConfig: [
+            {
+                op: "From Hex",
+                args: ["Space"]
+            },
+            {
+                op: "Extract EXIF",
+                args: [],
+            },
+        ],
+    },
+    {
+        name: "Extract EXIF: hello world text (error)",
+        input: "hello world",
+        expectedError: true,
+        recipeConfig: [
+            {
+                op: "From Hex",
```

Comment:
The input is not hex, so this recipe will return an error even if the 'Extract EXIF' operation doesn't throw one.